fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -872,7 +872,9 @@ private void parseDatatableColumnForAdd(final JsonObject column, StringBuilder s
             } else if (type.equalsIgnoreCase("Decimal")) {
                 sqlBuilder = sqlBuilder.append("(19,6)");
             } else if (type.equalsIgnoreCase("Dropdown")) {
-                sqlBuilder = sqlBuilder.append("(11)");
+                if (databaseTypeResolver.isMySQL()) {
```

Comment:
What happens for Postgres?

Reply:
Hi @hagafrank , the query will throw syntax error in Postgresql since INT can't used with length